### DISCUSSION TITLE
Is it possible to have more than one blog?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#108] [Ask a question] [answered]
Hi.

Situation: I have one site, but different blogs in one site, to separate themes from another.

I have played around with adjustments in the demo data, see if I can operate in a site multiple blogs side by side. However, functions for the tag filter, etc. collide. Is this solvable at a reasonable effort?

I have an alternative idea, so that's not a knock-out criteria, it interests me basically.

Regards,
NoSi

**Comment by markseuffert** [comment]
Yes, you can have multiple blogs on your website. Here is an example:
1. Download Yellow and duplicate the folder `content/2-blog`
2. Rename your new folder `content/2-articles` and open it
3. Edit your new `page.txt` and change `Title: Articles` at the top
4. Edit your blog example pages and add `Template: blog` to the top
5. That's it, you should have two different blogs now

Please add the template setting only to single blog pages, don’t add it to a page that already has one. 

How does it work? The template setting will tell Yellow what to do. If there is no template setting in a page, then Yellow will check if there is a template with the name of the current folder. This is what happens when you have a folder called `2-blog`, the blog template is automatically used. When you rename the blog folder to `2-articles`, then you need to tell manually which pages are blog articles. 

See also https://datenstrom.se/yellow/help/

Let me know if this works for you. Thanks.

**Comment by markseuffert** [comment, marked as answer]
Yes you can. Here's an updated example for Datenstrom Yellow 0.8.18

1. Download and unzip Yellow, select `Small blog` during installation
2. Duplicate the folder `content/2-blog` and name it `content/3-articles`
3. Edit file `content/3-articles/page.md` and change `Title: Articles` at the top

That's it, you should have two different blogs now.

**Comment by BsNoSi** [comment]
Hi Mark — this looks very interesting. I have a project in the near future I will try this technique. The usage of Yellow I decided weeks ago…  .;)
